src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visits/VisitsMigrationService.kt

```
@@ -147,7 +148,7 @@ class VisitsMigrationService(
         "nomisVisitId" to nomisVisitId.toString()
       ),
       null
-    ).also { throw e }
+    )
```

we swallow exception and manual procedure to fix this comes into operation